.gitignore

```diff
@@ -3,3 +3,4 @@
 /build/*
 /bin/*
 /phpunit.xml
+.idea
```

Comment:
This have nothing to be here. This is specific to your dev env.

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore